doc/Reference/Navigation/Navigation.md

```diff
@@ -1,6 +1,7 @@
 ---
 uid: Reference.Navigation.Overview
 ---
+
```

Comment:
@kazo0 Do we actually need this page? It is used as a Overview page for the reference docs:

https://github.com/unoplatform/uno.extensions/blob/46b2d7fc4834cefd77adc5d5fd1a0663815be83c/doc/toc.yml#L76

Reply:
We should have an Overview page that would open when clicking on the Navigation section in the TOC similar to what we do for MVUX in the Reference section: 

![image](https://github.com/user-attachments/assets/f87d4d5f-4165-4c45-92a3-a7d025a98cad)

Reply:
I would perhaps just use it as a landing page that could direct people to the individual pages in the Reference section for Navigation. Similar to this table: https://platform.uno/docs/articles/external/uno.extensions/doc/ExtensionsOverview.html